public/app/plugins/datasource/cloudwatch/components/ConfigEditor.tsx

```diff
@@ -115,6 +114,10 @@ export class ConfigEditor extends PureComponent<Props, State> {
     const { regions } = this.state;
     const { options } = this.props;
     const secureJsonData = (options.secureJsonData || {}) as CloudWatchSecureJsonData;
+    var profile = options.jsonData.profile;
+    if (!profile) {
+      profile = options.database;
```

Comment:
Should this be `options.jsonData.database`, since that's what the old code does?

Reply:
that was actually the error: the old code used `options.jsonData.database` instead of `options.database`.

Reply:
@papagian Ah... thanks for clarifying that!